test/parallel/test-fs-stream-options.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+require('../common');
+
+const assert = require('assert');
+const fs = require('fs');
+
+{
+  const fd = 'k';
+
+  assert.throws(
+    () => {
+      fs.createReadStream(null, { fd: fd });
+    },
+    {
+      code: 'ERR_INVALID_ARG_TYPE',
+      name: 'TypeError',
+    });
+
+  assert.throws(
+    () => {
+      fs.createWriteStream(null, { fd: fd });
```

Comment:
```suggestion
      fs.createWriteStream(null, { fd });
```

Reply:
Hi mscdex, thanks so much for your suggestions to improve my code. I have modified the original commit message as well as implemented the changes. I had some trouble with ESLint which unfortunately caused me to change/commit an unnecessary file. I have removed this file, and I am hoping things are in a good state right now, if not, apologies as I am still getting used to things. Let me know and I can make any changes asap.